multiple-domain/MultipleDomain.php

```diff
@@ -276,11 +335,17 @@ public function settingsFieldsForDomains()
      */
     public function settingsFieldsForOptions()
     {
-        $checked = $this->shouldIgnoreDefaultPorts() ? 'checked' : '';
-        echo '<label><input type="checkbox" name="multiple-domain-ignore-default-ports" value="1" ' . $checked . '> '
-            . __('Ignore default ports', 'multiple-domain') . '</label>'
-            . '<p class="description">' . __('When enabled, removes the port from URL when redirecting and '
-            . 'it\'s a default HTTP (<code>80</code>) or HTTPS (<code>443</code>) port.', 'multiple-domain') . '</p>';
+
+        echo '<label><input type="checkbox" name="multiple-domain-ignore-default-ports" value="1" ' . checked( $this->shouldIgnoreDefaultPorts(), true, false ) . '> ';
```

Comment:
This is similar to the comment right above. The text is being split in a way that can make translation difficult.